src/components/ClaimSearcher.js

```diff
@@ -290,10 +290,9 @@ class ClaimSearcher extends Component {
 
     let count = !!this.state.random && this.state.random.value;
     if (!count) {
-      count = claimsPageInfo.totalCount;
+      count = claimsPageInfo?.totalCount?.toString()?.replace(/\B(?=(\d{3})+(?!\d))/g, ",") ?? 0;
```

Comment:
toString() suggests that count shout be a string after this operation. In that case I think it should be ?? "0" not ?? 0.

Reply:
In that case it does not change anything. One line above, _count_ is defined as a number.

Reply:
As expected, because this code converts number to a string, there is no reason to assign a number type once again to it.

Reply:
What should be done here @wzglinieckisoldevelo  ?

Reply:
I think it should be changed to:
`      count = claimsPageInfo?.totalCount?.toString()?.replace(/\B(?=(\d{3})+(?!\d))/g, ",") ?? "0";`

The only difference being "0" instead of 0.

Reply:
LGTM now